crates/hyperdrive-math/src/lp.rs

```diff
@@ -353,6 +412,164 @@ impl State {
             ))
             .unwrap()
     }
+
+    /// Calculates the number of share reserves that are not reserved by open
+    /// positions.
+    fn calculate_idle_share_reserves(&self) -> FixedPoint {
+        let long_exposure = self.long_exposure().div_up(self.vault_share_price());
```

Comment:
I'm confused about this -- `self.long_exposure()` is in units of shares, right? So the variable you're creating is `shares/shares` and is unitless? I think that's fine, it happens, but it's not `long_exposure` anymore right? If so I think the variable name should change to something else.

Reply:
It must be in base and we're converting to shares.  This was translated from here:

https://github.com/delvtech/hyperdrive/blob/f76f5dacd38664cae6a2539770f30ff595d92141/contracts/src/internal/HyperdriveBase.sol#L575

Reply:
ahh ok, `z = x/c` so maybe just rename to `long_exposure_shares`?

Reply:
I think the name is right, marking as resolved.